src/assets/Azure.Core.Shared/OperationInternalBase.cs

```diff
@@ -58,6 +59,22 @@ protected OperationInternalBase(ClientDiagnostics clientDiagnostics, Response ra
         /// </summary>
         public TimeSpan DefaultPollingInterval { get; set; }
```

Comment:
I need more time to figure out if we can remove `DefaultPollingInterval`. Although it's public, it seems not be explicitly set (e.g. no place really set a non-default value) and the getter is invoked in polling implementation.
If we could remove it, then the logic of `PollingInterval` (see below) can be simplified.

Reply:
This type is defined in Azure.Core, so the change should happen there.